info.php

```diff
@@ -0,0 +1,3 @@
+<?php
+    phpinfo();
```

Comment:
May I know what's the use for the php info page?

Reply:
@Ninit29 
I forgot to delete the php-info file even though it was created for a temporary operation test. 
I deleted it, so please check again.